mycity/mycity/intents/get_alerts_intent.py

```diff
@@ -77,27 +77,69 @@ def get_alerts_intent(
     :param get_alerts_function_for_test: Injectable function for unit tests
     :param prune_normal_responses_function_for_test: Injectable function
      for unit tests
-    :param alerts_to_speech_output_function_for_test: Injectable function 
+    :param alerts_to_speech_output_function_for_test: Injectable function
     for unit tests
     :return: MyCityResponseDataModel object
     """
     logger.debug('MyCityRequestDataModel received:' +
                  mycity_request.get_logger_string())
 
-    alerts = get_alerts() if get_alerts_function_for_test is None \
-        else get_alerts_function_for_test()
-    logger.debug("[dictionary with alerts scraped from boston.gov]:\n" +
-                 str(alerts))
+    # get the intent_variables and sessions_attribute object from the request
+    intent_variables = mycity_request.intent_variables
+    session_attributes = mycity_request.session_attributes
+    decision = intent_variables['Decision'].get('value') \
```

Comment:
Similar to the slot name, I was confused by `decision` as a variable name. I think describing what the decision is, with something like `alert_type` is more clear